x-pack/test/api_integration/apis/management/cross_cluster_replication/remote_clusters.helpers.js

```diff
@@ -16,13 +16,16 @@ export const registerHelpers = (supertest) => {
   const addCluster = (name = REMOTE_CLUSTER_NAME) => {
     remoteClustersCreated.push(name);
 
+    // TODO how could this get pulled out of the ES config instead?
```

Comment:
Probably the only way would be to grab it from stdout during startup since it increments if the port isn't available.